recipes/typos/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
```

Comment:
Please also bundle third-party licenses.
```suggestion
cargo-bundle-licenses \
    --format yaml \
    --output THIRDPARTY.yml
```